src/Task/TaskFileSeeYou.hpp

```diff
@@ -26,11 +26,22 @@
 #include "TaskFile.hpp"
 #include "OS/Path.hpp"
 
+/**
+ * A class that reads and parses a SeeYou task file to an XCSoar internal
+ * task representation.
+ */
+
 class TaskFileSeeYou: public TaskFile
 {
 public:
-  explicit TaskFileSeeYou(Path _path):TaskFile(_path) {}
-
+  explicit TaskFileSeeYou(Path path):TaskFile(path) {}
+
+  /**
+   * Give the task produced by parsing the SeeYou file.
```

Comment:
Documentation belongs in a separate commit because it's unrelated to this new feature.